src/plugins/pluginManager/proposal.ts

```diff
@@ -88,6 +90,9 @@ export class PluginManagerProposal extends Proposal<IPluginManagerProposalState>
       pluginRegistered: item.schemeFactory.schemeRegistered,
       pluginRemoved: item.schemeFactory.schemeRemoved,
       pluginToRegisterData: item.schemeFactory.schemeToRegisterData,
+      pluginToRegisterDecodedData: item.schemeFactory.schemeToRemove === NULL_ADDRESS ?
+        pluginManager.decodeDataByPluginName(item.schemeFactory.schemeToRegisterName,
+          item.schemeFactory.schemeToRegisterData) : '',
```

Comment:
does `''` is empty object ?

Reply:
No it's empty string. I agree `{}` makes more sense.